docs/events/projections/healthchecks.md

```diff
@@ -0,0 +1,13 @@
+# Async Daemon HealthChecks
```

Comment:
**FIX:** Sounds like our spell checker doesn't know the word "healthcheck" (see https://github.com/JasperFx/marten/actions/runs/5848701384/job/16131086583?pr=2679). Could you add it to the config: https://github.com/JasperFx/marten/blob/master/docs/cSpell.json#L4?

Reply:
Sure - in the "words" section or "ignoreWords"?

Reply:
Yup 👍